mdoc/Test/en.expected-fsharp-wsl/Delegates+Delegate10.xml

```diff
@@ -1,5 +1,5 @@
 <Type Name="Delegates+Delegate10" FullName="Delegates+Delegate10">
-  <TypeSignature Language="C#" Value="public delegate char Delegates.Delegate10(int , int );" />
+  <TypeSignature Language="C#" Value="public delegate char Delegates.Delegate10(int unnamedParam1, int unnamedParam2);" />
```

Comment:
It's really strange that we're getting all these changes only now in this PR, when the unnamed parameter change happened all the way back in february with #535 🤔